use advent_of_code_2022::day10;

use anyhow::Error;

fn main() -> Result<(), Error> {
    day10::part2::solution()?;
    Ok(())
}
